configs/remediations.json

```diff
@@ -0,0 +1,22 @@
+{
+  "roles": [
+    {
+      "name": "Remediations Access",
+      "description": "A role which grants read and write permissions.",
+      "system": true,
+      "platform_default": true,
+      "version": 4,
+      "access": [
+        {
+          "permission": "remediations:remediation:read"
+        },
+        {
+          "permission": "remediations:remediation:write"
+        },
+        {
+          "permission": "inventory:*:*"
```

Comment:
Will remediations be making calls to inventory to need this implicit permission? Inventory does not yet implement RBAC, but this would account for it when they do.

Reply:
Yes, it's making calls to inventory to resolve host info based on host identifier.